### PR TITLE
Ref doc

### PR DESCRIPTION
**Pull request by Kantiran91**
I try to improve documentation and UX:
- put enduser information on TOP
- extend information
- add link to relevant doc in ReadMe.md

### CODE REVIEW

File: doc/Configuration.md [hunk 1]
```diff
@@ -31,6 +30,7 @@ The example `configuration.json` options are explained:
         {
           "Uri": "http://opcfoundation.org/UA/MachineTool/",
           "Id": "i=13",
+          "BaseTypeLevel": 0,  // ????
```
Comment (GoetzGoerisch) [inline doc/Configuration.md:33]: Can be removed?

File: doc/Configuration.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ The example `configuration.json` options are explained:
     "Password": "",
     "Prefix": "umati/v2", // Topic prefix
     "ClientId": "MyCompany/ClientName", // ClientId part of topic structure
-    "Protocol": "tcp", // tcp: plain; tls: TLS secured; wss: WebSocket TLS secured
+    "Protocol": "wss", // tcp: plain; tls: TLS secured; wss: WebSocket TLS secured
```
Comment (GoetzGoerisch) [inline doc/Configuration.md:60]: Port needs to be changed to 443 if wss here.

File: doc/Standalone.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-
-    Two Possibilities:
-    1. Stable version from [Releases](https://github.com/umati/Dashboard-OPCUA-Client/releases)
+1. Obtain the OPC UA Client Binaries:
+There are two options for obtaining the binaries:
+    1. Download the stable version from [Releases](https://github.com/umati/Dashboard-OPCUA-Client/releases)
 
         1. Select `latest` release.
         2. Download artefact for your operating system.
+           - use windows-2022 for most versions of windows 2010 and window 2011 (only older version need windows-2019 (see [#477](https://github.com/umati/Dashboard-OPCUA-Client/issues/477) for details)
```
Comment (GoetzGoerisch) [inline doc/Standalone.md:9]: Win 10 and Win 11

File: doc/Standalone.md [hunk 4]
```diff
@@ -34,6 +34,14 @@
...
     2. After the start up a JSON containing all machine values is published on the MQTT Broker at topic `umati/v2/<CompanyName>/<ClientName>/<SpecificationType>/<EscapedMachineNodeId>` (e.g. `umati/v2/ISW/ClientSampleServer/MachineToolType/nsu=http:_2F_2Fexample.com_2FBasicMachineTool_2F;i=66382`)
 
+## Installing the Debug version
+
+For the Debug version also the DLLs of Debugging are nessasry:
+
+- C++ 2013: <https://aka.ms/highdpimfc2013x86enu>
+- C++ 2022: <https://aka.ms/vs/17/release/vc_redist.x86.exeWindows10>
+- SDK: <https://go.microsoft.com/fwlink/p/?linkid=2196241>
+
```
Comment (GoetzGoerisch) [inline doc/Standalone.md:44]: @wlkrm or @marcfir this is for `release` or `debug` builds?
Included for `release` as far as I'm aware.

**Review by GoetzGoerisch — CHANGES_REQUESTED** [review]

File: doc/Configuration.md (hunk 1 above)
Comment (Kantiran91) [reply doc/Configuration.md:33]: after the discussion to day - Yes

File: doc/Configuration.md (hunk 2 above)
Comment (Kantiran91) [reply doc/Configuration.md:60]: yes

**Review by GoetzGoerisch — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**